python/cudf/cudf/core/column/timedelta.py

```diff
@@ -125,97 +151,105 @@ def to_pandas(
 
         return pd_series
 
-    def _binary_op_mul(self, rhs: BinaryOperand) -> DtypeObj:
-        if rhs.dtype.kind in ("f", "i", "u"):
+    def _binary_op_mul(self, other: ColumnBinaryOperand) -> DtypeObj:
+        if other.dtype.kind in ("f", "i", "u"):
             out_dtype = self.dtype
         else:
             raise TypeError(
-                f"Multiplication of {self.dtype} with {rhs.dtype} "
+                f"Multiplication of {self.dtype} with {other.dtype} "
                 f"cannot be performed."
             )
         return out_dtype
 
-    def _binary_op_mod(self, rhs: BinaryOperand) -> DtypeObj:
-        if pd.api.types.is_timedelta64_dtype(rhs.dtype):
-            out_dtype = determine_out_dtype(self.dtype, rhs.dtype)
-        elif rhs.dtype.kind in ("f", "i", "u"):
+    def _binary_op_mod(self, other: ColumnBinaryOperand) -> DtypeObj:
+        if pd.api.types.is_timedelta64_dtype(other.dtype):
+            out_dtype = determine_out_dtype(self.dtype, other.dtype)
+        elif other.dtype.kind in ("f", "i", "u"):
             out_dtype = self.dtype
         else:
             raise TypeError(
-                f"Modulus of {self.dtype} with {rhs.dtype} "
+                f"Modulus of {self.dtype} with {other.dtype} "
                 f"cannot be performed."
```

Comment:
```suggestion
                f"Modulo of {self.dtype} with {other.dtype} "
                f"cannot be computed."
```
The operation is named "modulo," while the result is called the "modulus." https://en.wikipedia.org/wiki/Modulo_operation